catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx

```diff
@@ -149,7 +149,7 @@ const SearchLayout: FunctionComponent<{ apiToolbarLinks: ApiToolbarLink[] }> =
             sortOrder: urlFormattedSort.sortOrder,
           }),
         },
-        pathname: pageLayoutMetadata.url.pathname,
+        pathname: router.pathname,
```

Comment:
I might be completely misunderstanding, but is an underlying issue here that the variables that are changing aren't in the `useEffect` dependencies?